pi-manage

```diff
@@ -1319,6 +1281,249 @@ def import_tokens(file, tokenrealm=None):
 token_manager.add_command('import', Command(import_tokens))
 
 
+def conf_export(conftype=DEFAULT_CONFTYPE_LIST, filename=None, name=None, stdout=True, print_passwords=False):
```

Comment:
the option exists only, since conf_export is called by fullexport and maybe you would like to have a plain output while you also get an archive file. I would leave it.

Reply:
I think our configuration is not that big that we need to output on both channels.
You can just call the export without a file and get the current config and if it looks good, just add the file-parameter to write it to file.

Reply:
I removed the extra stoud option.